test/ui-test/src/screens/StatusWelcomeScreen.py

```diff
@@ -222,24 +222,29 @@ def profile_settings_image_is_updated(self):
         # first time clicking on settings button closes the my profile modal
         click_obj_by_name(MainScreen.SETTINGS_BUTTON.value)
         click_obj_by_name(MainScreen.SETTINGS_BUTTON.value)
-        imagePresent("profiletestimage", True, 97, 100, 183, True)
+        myProfileSettingsObject = wait_and_get_obj(MainScreenComponents.PROFILE_SETTINGS_VIEW.value)
+        imagePresent("profiletestimage", True, 95, 100, 183, True, myProfileSettingsObject)
             
     def grab_screenshot(self):
         # take a screenshot of the profile image to compare it later with the main screen
         loginUserName = wait_and_get_obj(LoginView.LOGIN_VIEW_USER_IMAGE.value)
         grabScreenshot_and_save(loginUserName, "loginUserName", 200)
         
     def profile_image_is_updated(self):
-        profileNavBarButton = wait_and_get_obj(MainScreenComponents.PROFILE_NAVBAR_BUTTON.value)
-        imagePresent("loginUserName", True, 98, 85, 100, True)
-
+        # open profile popup and check image on profileNavBarButton and profileNavBarPopup
```

Comment:
Same here. Unrelated to your changes but it seems this method should be in StatusMainScreen instead.